app/scripts/lib/AccountIdentitiesPetnamesBridge.test.ts

```diff
@@ -181,13 +182,25 @@ describe('AccountIdentitiesPetnamesBridge', () => {
     );
   });
 
-  it('does not delete entry when account identity is deleted', () => {
+  it('does not delete non-participating Petname entries', () => {
```

Comment:
As we're already testing this with a mock implementation in the abstract, could we just test `isSyncParticipant` directly here to ensure it returns `true` or `false` based on the origin?